scripts/data_files/query_config.fmt

```diff
@@ -99,6 +99,10 @@
 #define MACRO_NAME_TO_STR(macro)                                        \
     mbedtls_printf( "%s", strlen( #macro "" ) > 0 ? #macro "\n" : "" )
 
+#define NAME_TO_STR(macro)  #macro
```

Comment:
Minor: that name is a bit misleading: `NAME_TO_STR(FOO)` expands to `"FOO"` but in `OUTPUT_MACRO_NAME_VALUE(macro)`, `NAME_TO_STR(macro)` expands to the expansion of `FOO`. I'm not sure a more generic name like `STRINGIFY` is necessarily better though.

Reply:
Changed to `STRINGIFY`